nss-ivory-trolls/BangazonSite
# set up scaffolding for controllers

# Description

Set up basic scaffolding for controllers.

## Type of change


# Testing Instructions for Change Made

1. `git fetch -a`
1. `git checkout cb-controllers-setup`
1. Check to see that views and controllers are scaffolded correctly

# Checklist:
